aspnetcore/fundamentals/openapi/openapi-comments.md

```diff
@@ -0,0 +1,307 @@
+* Response documentation with status codes and descriptions.
+* Support for examples and deprecated markers.
+
+The `XmlComment` class processes [XML documentation tags](https://learn.microsoft.com/dotnet/csharp/language-reference/xmldoc/recommended-tags) like: `<c>`, `<code>`, `<list>`, `<para>`, `<paramref>`, `<typeparamref>`, `<see>`, and `<seealso>`. For XML documentation tags that use references to other elements, like `<see cref="SomeOtherType">`, the implementation strips out the XML tag and maps the reference to plain text for inclusion in the OpenAPI document.
+
+### Support for `<inheritdoc/>`
+
+<!-- original - review my change 
+`<inheritdoc />` tags present an important opportunity because they indicate that comments must be resolved from a base class or implemented interface. 
+The source generator uses its knowledge of the symbol's present in the compilation to discover base classes and interfaces associated with the symbol a given `<inheritdoc />` is placed on and supports resolving them automatically
+-->
+
+`<inheritdoc />` tags indicate that comments must be resolved from a base class or implemented interface. When `<inheritdoc />` is placed on a symbol, The source generator:
+
+* Uses its knowledge of the symbol in the compilation to discover base classes and interfaces associated with the symbol.
+* Supports resolving them automatically
+
+The automatic resolution behavior is currently available for XML documentation comments that exist in the assembly under compilation, and ***not*** XML documentation tags that are in referenced projects or packages. In the later scenario, XML documentation comments:
+
+* Are only presented as text.
+* There's no trivial strategy for:
+  * Associating the text content to compilation symbols.
+  * Developing an understanding of the inheritance hierarchy associated with the types.
+
+### Member Identification
+
+The source generator discovers XML comments statically and emits code that applies them to the document dynamically at runtime. The [`MemberKey`](https://source.dot.net/#Microsoft.AspNetCore.OpenApi.SourceGenerators/XmlComments/MemberKey.cs,d182ed147edb11d2) class acts as a bridge between compile-time and runtime representations of the same concept. It's a unique identifier for types, methods, and properties that works across:
+
+* Different compilation environments.
+* Generic types with proper handling of open generics.
+* Method overloads with parameter signature matching.
+
+The `MemberKey` definition attempts to encode as much information as possible to map a compile-time symbol to its runtime counterpart.
+
+```csharp
+internal sealed record MemberKey(
+    string? DeclaringType,
+    MemberType MemberKind,
+    string? Name,
+    string? ReturnType,
+    string[]? Parameters) : IEquatable<MemberKey>
+```
+
+### Code Generation
+
+The generator emits code that contains:
+
+* A cache of XML comments mapped to member identifiers:
```

Comment:
I don't think we need to provide an example of the generated code here since it's an implementation detail. Just stating that a cache is used is sufficient, IMO.